src/main/java/com/solace/samples/jms/snippets/HowToImplementManualInstrumentation.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.solace.samples.jms.snippets;
+
+import com.solace.opentelemetry.javaagent.jms.SolaceJmsW3CTextMapGetter;
+import com.solace.opentelemetry.javaagent.jms.SolaceJmsW3CTextMapSetter;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.api.trace.StatusCode;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.context.propagation.TextMapPropagator;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes.MessagingDestinationKindValues;
+import io.opentelemetry.semconv.trace.attributes.SemanticAttributes.MessagingOperationValues;
+import java.util.function.Consumer;
+import javax.jms.Message;
+import javax.jms.MessageProducer;
+import javax.jms.Topic;
+
+/**
+ * Example how to manually instrument Solace JMS publisher and receiver. All examples assume usage
+ * of a newer Solace JMS API v10.18.0 or newer.
+ */
+public class HowToImplementManualInstrumentation {
```

Comment:
Please rename to `HowToImplementTracingManualInstrumentation` to be specific that this is about tracing like we did for JCSMP as well.